examples/admin_policy/example_server/policy_server.py

```diff
@@ -20,39 +21,24 @@ async def apply_policy(request: Request) -> JSONResponse:
     # Decode from request body
     json_data = await request.json()
     user_request = sky.UserRequest.decode(json_data)
+    # Change the following list to apply one or more policies to the user
+    # request. You can also implement custom policies in server in addition to
+    # loading policies from external packages.
+    policies: List[sky.AdminPolicy] = [
+        example_policy.SetMaxAutostopIdleMinutesPolicy,
+        example_policy.UseSpotForGpuPolicy,
```

Comment:
Could we still keep these policies inline to make it easier to read and setup?

Reply:
instead of requiring a user to install `example_policy` as a package, just use it as a class in this file.